docs/content/docs/integrations/aws.md

```diff
@@ -480,7 +480,7 @@ AWS_PACKAGES=(
 )
 
 ICEBERG_PACKAGES=(
-  "iceberg-spark-runtime-3.2_2.12"
+  "iceberg-spark3-runtime"
```

Comment:
Looks like these changes were made here but not synced back in `apache/iceberg`. @jackye1995 should we do one more manual resolution and try to get a consensus on a contributing pattern?

Reply:
yes agree, let's do manual resolutions for these for 0.13.1, and let me start a thread for doc change contribution pattern in the dev list